apollo-backend/src/test/java/io/logz/apollo/SingleRegionBlockerTest.java

```diff
@@ -0,0 +1,98 @@
+package io.logz.apollo;
+
+import io.logz.apollo.clients.ApolloTestAdminClient;
+import io.logz.apollo.clients.ApolloTestClient;
+import io.logz.apollo.helpers.Common;
+import io.logz.apollo.helpers.ModelsGenerator;
+import io.logz.apollo.models.BlockerDefinition;
+import io.logz.apollo.models.DeployableVersion;
+import io.logz.apollo.models.DeploymentPermission;
+import io.logz.apollo.models.Environment;
+import io.logz.apollo.models.MultiDeploymentResponseObject;
+import io.logz.apollo.models.Service;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static io.logz.apollo.helpers.ModelsGenerator.createAndSubmitBlocker;
+import static io.logz.apollo.helpers.ModelsGenerator.createAndSubmitDeployableVersion;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SingleRegionBlockerTest {
+    private final static String BLOCK_TYPE_NAME = "singleregion";
+
+    @Test
+    public void testSingleRegionBlockerWithMultiEnvironmentsException() throws Exception {
+        ApolloTestClient apolloTestClient = Common.signupAndLogin();
+        ApolloTestAdminClient apolloTestAdminClient = Common.getAndLoginApolloTestAdminClient();
+
+        Service serviceToBeLimitToOneRegion = ModelsGenerator.createAndSubmitService(apolloTestClient);
+        DeployableVersion deployableVersion = ModelsGenerator.createAndSubmitDeployableVersion(apolloTestClient, serviceToBeLimitToOneRegion);
+
+        List<Integer> serviceIds = new ArrayList<Integer>() {{ add(serviceToBeLimitToOneRegion.getId()); }};
+        BlockerDefinition blocker = createAndSubmitBlocker(apolloTestAdminClient, BLOCK_TYPE_NAME, getSingleRegionBlockerConfiguration(serviceIds), null, null, null, null);
+
+        Environment env1 = ModelsGenerator.createAndSubmitEnvironment(apolloTestClient);
+        Environment env2 = ModelsGenerator.createAndSubmitEnvironment(apolloTestClient);
+
+        String envIdsCsv = String.valueOf(env1.getId()) + "," + String.valueOf(env2.getId());
+
+        MultiDeploymentResponseObject result = apolloTestClient.addDeployment(envIdsCsv, String.valueOf(serviceToBeLimitToOneRegion.getId()), deployableVersion.getId());
+        assertThat(result.getUnsuccessful().get(0).getException().getMessage().contains("you can not deploy the next service on multi environments"));
+
+
+        blocker.setActive(false);
+        apolloTestAdminClient.updateBlocker(blocker);
```

Comment:
should be done in `@AfterEach`

Reply:
Done.

Reply:
@matvey-mtn we need to upgrade the junit version for the project if you want to enable the AfterEach annotation in our tests.

Reply:
you can use `org.junit.After`

Reply:
Done.